### PR TITLE
Upgrade to v1.3.3, patch change_url

### PR DESCRIPTION
**Pull request by tituspijean**

### CODE REVIEW

**Comment by tituspijean** [comment]
!testme

File: scripts/change_url [hunk 1]
```diff
@@ -8,6 +8,7 @@
 
 source _common.sh
 source /usr/share/yunohost/helpers
+ynh_abort_if_errors
```
Comment (ericgaspar) [inline scripts/change_url:11]: you can instead integrate the backup before change url part...
https://github.com/YunoHost/example_ynh/blob/b86f6da6b5ca3d905297af4d5ce98cf4ca8a217e/scripts/change_url#L37-L53

Comment (tituspijean) [reply scripts/change_url:11]: Buuuut it was suggested by an autopatch (https://github.com/YunoHost-Apps/nodered_ynh/pull/22). Though I agree, I should follow the example app standards.

**Comment by tituspijean** [comment]
!testme

**Review by alexAubin — APPROVED** [review]
Quick review, LGTM

**Comment by tituspijean** [comment]
!testme

**Review by ericgaspar — APPROVED** [review]

**Pull request merged**